pkg/crc/machine/machine.go

```diff
@@ -204,6 +206,12 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		return *result, err
 	}
 
+	// Check the certs validity inside the vm
+	logging.Info("Verifying validity of the cluster certificates ...")
+	if err := checkClusterCertsValidity(host.Driver); err != nil {
+		result.Error = err.Error()
+		return *result, err
+	}
```

Comment:
@anjannath this block should go https://github.com/code-ready/crc/pull/310/files#diff-06d0e75b8f65eb58c0f83515f2a7217dL146 here, we don't want to execute anything on the VM if certs are not valid.